packages/react-graphql-universal-provider/src/GraphQLUniversalProvider.tsx

```diff
@@ -6,14 +6,17 @@ import {
   createSsrExtractableLink,
   createOperationDetailsLink,
 } from '@shopify/react-graphql';
+import ApolloClient, {ApolloClientOptions} from 'apollo-client';
 import {useLazyRef} from '@shopify/react-hooks';
 
+export type GraphQLClientOptions = ApolloClientOptions<any>;
+
 interface Props {
   children?: React.ReactNode;
-  createClient(): import('apollo-client').ApolloClient<any>;
+  clientOptions: ApolloClientOptions<any>;
```

Comment:
I think we actually want this to be `clientOptions(): ApolloClientOptions<any>`, for the same reasons the previous api was `createClient(): ApolloClient` not  `client: ApolloClient`.